### ISSUE TITLE
Negative potential radiation (Qmax)

### ISSUE THREAD

**Issue by femiguez** [APSIMInitiative/APSIMClassic#1803]
With some met files, potential radiation (Qmax) is negative. Perhaps a warning could be thrown in this case. Here is an example file
[cia-daymet.met.zip](https://github.com/APSIMInitiative/APSIMClassic/files/4177820/cia-daymet.met.zip)
This happens with APSIM-X and 'Classic'. Not sure what the source of this is.

**Comment by hol430** [comment]
Hi @femiguez, it looks like your vapor pressure is in pascals, but next gen expects this in hPa. No idea about old apsim, but I would hazard a guess that it also expects VP in hpa.

**Comment by femiguez** [comment]
Hi @hol430 ! Thanks so much! Can you point me to the file that processes the met file in Next Gen? It is not intuitive to me that VP is used in the calculation of maxQ and I'm wondering what else is VP used for in the model. This does in fact resolve the issue.

**Comment by hol430** [comment]
Yep, in next gen it's calculated in [ApsimX/APSIM.Shared/Utilities/MetUtilities.cs:324](https://github.com/APSIMInitiative/ApsimX/blob/d9c20e447cb87997d1f28be6267ad1170495122e/APSIM.Shared/Utilities/MetUtilities.cs#L324) in the function QMax. If you're using a standard met component this will be called from [ApsimX/Models/Weather/Weather.cs:665](https://github.com/APSIMInitiative/ApsimX/blob/d9c20e447cb87997d1f28be6267ad1170495122e/Models/Weather/Weather.cs#L665).
